[2022-05-22 09:35:48,695] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: backup_all_db.load_csv_to_new_db scheduled__2022-05-21T00:00:00+00:00 [queued]>
[2022-05-22 09:35:48,706] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: backup_all_db.load_csv_to_new_db scheduled__2022-05-21T00:00:00+00:00 [queued]>
[2022-05-22 09:35:48,706] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 09:35:48,706] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-05-22 09:35:48,706] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 09:35:50,081] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): load_csv_to_new_db> on 2022-05-21 00:00:00+00:00
[2022-05-22 09:35:50,130] {standard_task_runner.py:52} INFO - Started process 78410 to run task
[2022-05-22 09:35:50,135] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'backup_all_db', 'load_csv_to_new_db', 'scheduled__2022-05-21T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmp2f2dv1vd', '--error-file', '/tmp/tmp7vibj3nt']
[2022-05-22 09:35:50,136] {standard_task_runner.py:80} INFO - Job 10: Subtask load_csv_to_new_db
[2022-05-22 09:35:50,977] {task_command.py:369} INFO - Running <TaskInstance: backup_all_db.load_csv_to_new_db scheduled__2022-05-21T00:00:00+00:00 [running]> on host homerim-desktop
[2022-05-22 09:35:51,554] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=backup_all_db
AIRFLOW_CTX_TASK_ID=load_csv_to_new_db
AIRFLOW_CTX_EXECUTION_DATE=2022-05-21T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-21T00:00:00+00:00
[2022-05-22 09:35:51,555] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-22 09:35:51,556] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python3 ~/airflow/dags/tasks/load_csv_to_new_db.py']
[2022-05-22 09:35:51,566] {subprocess.py:85} INFO - Output:
[2022-05-22 09:37:12,722] {subprocess.py:92} INFO - В новую базу загружен файл orders
[2022-05-22 09:37:12,723] {subprocess.py:92} INFO - В новую базу загружен файл supplier
[2022-05-22 09:37:12,723] {subprocess.py:92} INFO - В новую базу загружен файл nation
[2022-05-22 09:37:12,723] {subprocess.py:92} INFO - В новую базу загружен файл lineitem
[2022-05-22 09:37:12,723] {subprocess.py:92} INFO - В новую базу загружен файл region
[2022-05-22 09:37:12,724] {subprocess.py:92} INFO - В новую базу загружен файл customer
[2022-05-22 09:37:12,724] {subprocess.py:92} INFO - В новую базу загружен файл partsupp
[2022-05-22 09:37:12,724] {subprocess.py:92} INFO - В новую базу загружен файл part
[2022-05-22 09:37:12,724] {subprocess.py:92} INFO - Все данные успешно перенесены!
[2022-05-22 09:37:12,724] {subprocess.py:92} INFO - В таблице customer находится 150000 записей
[2022-05-22 09:37:12,724] {subprocess.py:92} INFO - В таблице lineitem находится 6001215 записей
[2022-05-22 09:37:12,724] {subprocess.py:92} INFO - В таблице partsupp находится 800000 записей
[2022-05-22 09:37:12,725] {subprocess.py:92} INFO - В таблице nation находится 50 записей
[2022-05-22 09:37:12,725] {subprocess.py:92} INFO - В таблице orders находится 1500000 записей
[2022-05-22 09:37:12,725] {subprocess.py:92} INFO - В таблице region находится 5 записей
[2022-05-22 09:37:12,725] {subprocess.py:92} INFO - В таблице supplier находится 20000 записей
[2022-05-22 09:37:12,725] {subprocess.py:92} INFO - В таблице part находится 200000 записей
[2022-05-22 09:37:12,730] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-22 09:37:13,445] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=backup_all_db, task_id=load_csv_to_new_db, execution_date=20220521T000000, start_date=20220522T063548, end_date=20220522T063713
[2022-05-22 09:37:13,490] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-22 09:37:13,509] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
